open-subshare/REPWRITERSPECS/BANNO.NEWSUBCREATE.V1.POW

```diff
@@ -1,7 +1,7 @@
 [*
 **  PowerOn Name:       BANNO.NEWSUBCREATE.V1.POW
 **  Datafile Name:      BANNO.NEWSUBCREATE.V1.CFG
-**  Letterfile Name:    BANNO.NEWSUBCREATE.TERMS.00-09
+**  Letterfile Name:    BANNO.NEWSUBCREATE.TERMS.00-19
 **
 **  Copyright 2020-2022 Jack Henry and Associates
 **
```

Comment:
I was a little thrown off at at line 2100, as there is the only remaining "CALL JSONCLOSE"
in the program. I think it might be correct to put it there, as you do another 
open bracket after that and then do PRINTPROGRAMINFO.

I had just noticed you removed all the other "CALL JSONCLOSE" parts of the code as 
you have added the following to the end of the
print division, PRINT BNOINDENT(0)+"}" .

Reply:
Yes, instead of having a JSONCLOSE at the end of each (many - actually) procedures, it was just added to the end of the print division. As for the JSONCLOSE calls, there are two remaining and thus the procedure call instead of imbedding the actual JSON closing brace into the code directly.